dependency-check/global-suppressions.xml

```diff
@@ -24,7 +24,7 @@
     <notes><![CDATA[
    This is a low severity (<4) vuln with no fix released supported in jetty 9. Revisit on jetty upgrade.
    ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.eclipse\.jetty/jetty\-http@9.*$</packageUrl>
+    <packageUrl regex="true">^pkg:maven/org\.eclipse\.jetty/jetty.*@9.*$</packageUrl>
```

Comment:
The reasoning is the same. NVD for some strange reason changed the CPE match to flag all jetty packages even though the minor issue only appears in jetty-http.